tests/kernel/kdump.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Configure and run kdump test using command line interface
+# Maintainer: QE Kernel <kernel-qa@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use kdump_utils;
+
+sub run {
+    my ($self) = @_;
+    select_console('root-console');
+    kdump_utils::configure_service('function', yast_interface => 'cli');
```

Comment:
We don't really need `kdump_utils::` as you did the import above.

Reply:
Another minor point, as you already accept hash in the method, I would propose to convert 'function' string to the boolean key-pair and set `function => 1`

Reply:
> Another minor point, as you already accept hash in the method, I would propose to convert 'function' string to the boolean key-pair and set `function => 1`

But this would mean to rewrite  "function" handling for all migration tests and other uses.

Reply:
It might sound like much, but it's safe and easy change. Anyway, it's not that we have to do it now.

Reply:
> It might sound like much, but it's safe and easy change. Anyway, it's not that we have to do it now.

Other parts of the code use `configure_service('before')`, which means that change needs better analysis (not just migrate `function` to  boolean, but maybe to `test_type=> 'function'`) I will create poo and place it to "Migration" topic.

Reply:
>  Anyway, it's not that we have to do it now.
Will be done in https://progress.opensuse.org/issues/88519